examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal

```diff
@@ -11,88 +11,67 @@ type Student record {
 
 public function main() returns error? {
     // Runs the prerequisite setup for the example.
-    check beforeExample();
+    check initialize();
 
-    // Initializes the MySQL client.
-    mysql:Client mysqlClient = check new (user = "root", 
-            password = "Test@123", database = "MYSQL_BBE");
+    // Initializes the MySQL client. The `mysqlClient` can be reused to access database throughout the application execution.
+    mysql:Client mysqlClient = check new (user = "root",
+            password = "Test@123", database = "STUDENT");
 
-    // Creates a parameterized query to invoke the procedure.
-    string name = "George";
-    int age = 24;
-    sql:ParameterizedCallQuery sqlQuery = 
-                                `CALL InsertStudent(${name}, ${age})`;
-
-    // Invokes the stored procedure `InsertStudent` with the `IN` parameters.
-    sql:ProcedureCallResult retCall = check mysqlClient->call(sqlQuery);
-    io:println("Call stored procedure `InsertStudent`." + 
-        "\nAffected Row count: ", retCall.executionResult?.affectedRowCount);
-    check retCall.close();
-
-    // Initializes the `INOUT` and `OUT` parameters.
+    // Initializes the `INOUT` and `OUT` parameters for the procedure call.
     sql:InOutParameter id = new (1);
     sql:IntegerOutParameter totalCount = new;
-    sql:ParameterizedCallQuery sqlQuery2 = 
-                        `{CALL GetCount(${id}, ${totalCount})}`;
-
-    // The stored procedure with the `OUT` and `INOUT` parameters is invoked.
-    sql:ProcedureCallResult retCall2 = check mysqlClient->call(sqlQuery2);
-    io:println("Call stored procedure `GetCount`.");
-    io:println("Age of the student with id '1' : ", id.get(int));
-    io:println("Total student count: ", totalCount.get(int));
-    check retCall2.close();
-
-    // Invokes the stored procedure, which returns the data.
-    sql:ProcedureCallResult retCall3 = 
-            check mysqlClient->call(`{CALL GetStudents()}`, [Student]);
-    io:println("Call stored procedure `GetStudents`.");
-
-    // Processes the returned result stream.
-    stream<record {}, sql:Error?>? result = retCall3.queryResult;
-    if result is stream<record {}, sql:Error?> {
-        stream<Student, sql:Error?> studentStream =
-                <stream<Student, sql:Error?>>result;
-        check from Student student in studentStream
-            do {
-                io:println("Student details: ", student);
-            };
-    }
-    check retCall3.close();
+
+    // The stored procedure is invoked.
+    sql:ProcedureCallResult result =
+        check mysqlClient->call(`{CALL GetCount(${id}, ${totalCount})}`);
+
+    // Process procedure call parameters.
```

Comment:
```suggestion
    // Process procedure for the call parameters.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)